src/main/java/com/epam/ta/reportportal/ws/param/HistoryRequestParams.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2019 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.epam.ta.reportportal.ws.param;
```

Comment:
it's only for history - so probably it is good idea to put it near history logic and split to different classes and give them docs

Reply:
Moved to another package. Java docs provided